### PR TITLE
skeleton of the command to load scripts

### PR DESCRIPTION
**Pull request by gogo2464**
This PR does not do nothing interesting for the moment. It is just to show. Next I will implement the run: [see](https://github.com/Rair-Project/rair-core/compare/master...gogo2464:gogo/2_Add-command-for-loading-script-files?expand=1#diff-df186358b1dffce0133defc6b9ef7019R35).

I should have done:

`core.add_command("executeScript", "es", Arc::new(Mutex::new(ExecuteScript:new())));` instead. I will fix it as soon as possible.

### CODE REVIEW

**Comment by gogo2464** [comment]
I need to update the [unit test](https://travis-ci.com/github/Rair-Project/rair-core/jobs/382108999#L494) to add my new command.

File: src/utils/cmd.rs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+ * cmd.rs: commands for handling another commands.
+ * Copyright (C) 2019  Oddcoder
```
Comment (oddcoder) [inline src/utils/cmd.rs]: I bet I didn't write this file!

Comment (gogo2464) [reply src/utils/cmd.rs]: Ah yes! I did not change the name yet.

**Comment by gogo2464** [comment]
I added new unit test to make coverage be better.

**Pull request closed without merging**